pkg/report/writer.go

```diff
@@ -106,8 +106,8 @@ func (tw TableWriter) write(result Result) {
 	}
 
 	var results []string
-	for _, severity := range dbTypes.SeverityNames {
-		r := fmt.Sprintf("%s: %d", severity, severityCount[severity])
+	for _, severity := range severities {
```

Comment:
I think we have to uniq/sort `severities`. I'm worried that a user specifies `--severity HIGH,HIGH,CRITICAL` or `--severity MEDIUM,LOW,HIGH`.